seed/seed20.go

```diff
@@ -762,3 +763,34 @@ func (s *seed20) Iter(f func(sn *Snap) error) error {
 	}
 	return nil
 }
+
+func (s *seed20) LoadAutoImportAssertions(commitTo func(*asserts.Batch) error) {
+	if s.model.Grade() == asserts.ModelDangerous {
```

Comment:
Maybe a comment here why we limit loading assertions to grade dangerous and if we ever plan to change/rethink this?

Reply:
I think we should keep it dangerous only for now and document this. The other use cases wouldn't be supported this way anyway, they probably would involve assertions that go in the main assertion directory.